### ISSUE TITLE
Cannot view Syllabus in Practice Mode

### ISSUE THREAD

**Issue by UltraGamerine** [exercism/exercism#6609]
We should be able to read and learn from any Topic and solve any problem in Practice mode. People who already have some idea of the language may not want to go through the basics, but may want to read advance stuff. I could View and Read any topic before I signed it, but after signing in I can only view just the description of the Topics in Learning Mode, or no Syllabus tab in Practice mode, which is a big bummer.

**Comment by MaxLap** [comment]
I totally agree here. This is a frustrating experience. Many dialogs tell you that you have unlocked concepts, maybe one of them is interesting since but clicking "view concepts" button in the modals bring your to the list of all exercises (overview). My only option then is to go to exercices, do a ctrl+f hoping the concept I saw shows up in the short description.

I have a good clue of what I want to go through and what I want to skip. But I don't know the optimal order to go through them, especially considering previous concepts are often reused. As an example, I was doing type defs without having seen tuples yet or maps yet, oops. The sillabus (when I turn learning mode back on) would have easily shown me that tuples should come first by just doing things that interest me from the top.

The syllabus appears to work just fine. When I toggle learning mode on and off, it takes takes for exercises to lock / unlock. But access to syllabus is instant. So by toggling, frequently toggling, I can kinda use the syllabus, with every exercises unlocked. Exactly what I want.

So syllabus being locked in practice mode feels like an arbitrary restriction more than a technical one.
